frontend/src/components/productCard/ProductCard.tsx

```diff
@@ -44,18 +44,27 @@ export const ProductCard = ({ item }: ProductCardProps) => {
                                 </button>
                             </div>
                             <div className={styles.placeBidContainer}>
-                                <button type="button" className={styles.placeBidButton}>
-                                    <span>Place a bid</span>
-                                </button>
+                                {item.instantSellPrice === true && (
```

Comment:
tip: you don't have to explicitly compare to `true` unless the value of `instantSellPrice` is not a boolean (just checking if the value is truthy should be enough in this case - `item.instantSellPrice && ()`)